client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SearchDocumentationIT.java

```diff
@@ -100,24 +101,26 @@ public void testScroll() throws IOException {
         }
         {
             String scrollId = lastScrollId;
-
-            // tag::search-scroll-id
-            SearchScrollRequest scrollRequest = new SearchScrollRequest(); // <1>
-            scrollRequest.scroll(TimeValue.timeValueMinutes(1L)); // <2>
-            scrollRequest.scrollId(scrollId); // <3>
-            // end::search-scroll-id
-
             // tag::search-scroll-execute
-            SearchResponse searchResponse = client.searchScroll(scrollRequest); // <1>
-            SearchHits searchHits = searchResponse.getHits();  // <2>
-            String nextScrollId = searchResponse.getScrollId();  // <3>
+            while (true) {
```

Comment:
Ok, I updated the doc in https://github.com/elastic/elasticsearch/pull/25554/commits/0c2d8152f52d3eb91b1cb63a249f56b671f879ec, let me know if this is OK or not. 

I'm personally fine with the latest version as well as the previous one too, just let me know which one seems better to you.